README.md

```diff
@@ -1,50 +1,214 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
 
-# Microverse README Template
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-## 📗 Table of Contents
+Modify this file to match your project and remove sections that don't apply.
 
-> 📖 About the Project
-> 🛠 Built With
-> HTML
-> CSS
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-## 🚀 Live Demo
-https://abenezerassefa.github.io/
+OPTIONAL SECTIONS:
+- FAQ
 
-## 💻 Getting Started
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
- [ OPTIONAL] Please remove this comment after editing your README. Line 25 of the comments gives it all away 😆